.github/workflows/ci_windows_x64_msvc.yml

```diff
@@ -22,19 +22,22 @@ concurrency:
 
 jobs:
   windows_x64_msvc:
-    runs-on: windows-2022
-    defaults:
-      run:
-        shell: bash
+    runs-on: arc-runner-set
```

Comment:
Please share logs of this workflow running.

The current triggers aren't running this workflow on your pull request:
```yml
on:
  schedule:
    # Weekday mornings at 09:15 UTC = 01:15 PST (UTC - 8).
    - cron: "15 9 * * 1-5"
  workflow_dispatch:
```

After this lands we should be able to switch that to run on `pull_request` and `push` triggers (see https://github.com/iree-org/iree/issues/18813)

(I'll have other comments but first thing need to see how long this takes to run, the breakdown step by step, and other logs)

Reply:
how can I run the workflow before its merged in? should i switch this PR to merge into a branch, so I can run it through actions, or is it possible to run actions through a fork

Reply:
A bunch of different options:

* Switch the trigger to `pull_request` in your PR and iterate here
* Use `workflow_dispatch` by pushing to a branch in the shared repo here and running via the "Run workflow" button on https://github.com/iree-org/iree/actions/workflows/ci_windows_x64_msvc.yml
* Switch the trigger to `push` in your fork and just push to a branch, without sending a pull request

Working in your fork or using `workflow_dispatch` keeps alert volume down (emails for users subscribed to the repo).